web/cobrands/fixmystreet-uk-councils/council_category_hide_element_rules.js

```diff
@@ -0,0 +1,8 @@
+hide_element_rules = {
+    'Central Bedfordshire Council': {
+        'Fly Tipping': [
+            "description-tips",
```

Comment:
If this was a selector that could be passed straight to `$` on L692 of `web/cobrands/fixmystreet/fixmystreet.js` it'd be more flexible than only allowing element IDs.